frontstage/templates/help/help-with-password-not-accept.html

```diff
@@ -0,0 +1,36 @@
+{% extends "layouts/_block_content.html" %}
+{% from "components/breadcrumb/_macro.njk" import onsBreadcrumb %}
+{% from "components/button/_macro.njk" import onsButton %}
+
+{% set page_title = "Help password not accepted" %}
+{% set breadcrumbsData = [
+  {
+    "text": "Sign in",
+    "url": "/sign-in"
+  },
+  {
+    "text": "Help",
+    "url": "/help"
+  },
+  {
+    "text": "Help with my password",
+    "url": "/help/help-with-my-password"
+  }
+] %}
+{% block breadcrumb %}
+{{
+    onsBreadcrumb({
+        "ariaLabel": "Breadcrumbs",
+        "id": "breadcrumbs",
+        "itemsList": breadcrumbsData
+    })
+}}
+{% endblock breadcrumb %}
+
+{% block main %}
+<h1 class="u-fs-xl">My new password isn't being accepted</h1>
+<p>You cannot include the special character "@" or "at" symbol, in your password.</p>
```

Comment:
is this content right?

Reply:
I believe so.. it went via the design review.